src/guardian-hosted/web/index.scss

```diff
@@ -0,0 +1,251 @@
+    color: #333333;
+    margin: 0;
+    text-align: center;
+    .hosted-bright & {
+        color: #ffffff;
+    }
+}
+
+.hostedbadge__logo {
+    width: 100%;
+    display: block;
+}
+
+.hosted-trafficdriver__tile {
+    text-decoration: none;
+    display: inline-block;
+    position: relative;
+    flex: 1;
+
+    @include mq($until: mobileLandscape) {
+        margin-left: 0;
+
+        &:not(:first-child) {
+            margin-top: 10px;
+            .hosted-trafficdriver__image {
+                display: none;
+            }
+        }
+    }
+
+    @include mq(mobileLandscape, tablet) {
+        margin-left: 0;
+
+        &:first-child {
+            margin-right: 20px;
+        }
+
+        &:nth-child(n+3) {
+            margin-top: 10px;
+            flex-basis: 100%;
+            flex-grow: 0;
+            flex-shrink: 0;
+
+            .hosted-trafficdriver__image{
+                display: none;
+            }
+        }
+    }
+
+    @include mq(tablet) {
+        &:not(:first-child) {
+            margin-left: 20px;
+        }
+    }
+
+    .hosted-trafficdriver__image{
+        background-size: cover;
+        background-position: center;
+
+        &:before {
+            content: '';
+            padding-top: 100%;
+            display: block;
+        }
+    }
+
+    .with-2-elements &:nth-child(3),
+    .with-2-elements &:nth-child(4),
+    .with-3-elements &:nth-child(4) {
+        display: none;
+    }
+}
+
+@include mq(desktop) {
+    .with-2-elements {
+        .hosted-trafficdriver__image {
+            width: 50%;
+        }
+        .hosted-trafficdriver__title {
+            position: absolute;
+            top: 0;
+            left: 50%;
+            font-size: 18px;
+        }
+    }
+}
+
+.hosted-trafficdriver__title {
+    padding: 6px 6px 18px;
+    font-size: 16px;
+    line-height: 22px;
+    font-weight: 600;
+    color: #333333;
+    .hosted-bright & {
+        color: #ffffff;
+    }
+
+    @include mq($until: desktop) {
+        font-size: 20px;
+        line-height: 24px;
+    }    
+}
+
+.hosted-trafficdriver__content-type {
+    opacity: 0.5;
+}
+
+.hosted-trafficdriver__body {
+    position: relative;
+    display: flex;
+
+    @include mq($until: mobileLandscape) {
+        flex-direction: column;
+    }
+
+    @include mq(mobileLandscape) {
+        flex-flow: row wrap;
+    }
+
+    @include mq(leftCol) {
+        margin-left: 160px;
+    }
+
+    @include mq(wide) {
+        margin-left: 240px;
+        width: 940px;
+    }
+}
+
+.fc-container__header__title {
+    float: left;
+    font-size: 22px;
+    font-weight: 600;
+    color: #333333;
+    line-height: 22px;
+    margin: 0;
+    
+    @include mq($until: leftCol){
+        font-size: 20px;
+        line-height: 24px;
+    }
+
+    @include mq(leftCol){
+        width: 140px;
+    }
+}
+
+@mixin side-margins-position($container-width) {
+    // stupid fix for sasslint AST error
+    $size: $container-width + 40px;
+
+    &:before,
```

Comment:
for pseudo-elements, you need to use `::`, e.g. `::after`. One colon is for pseudo-_selectors_ and is deprecated for pseudo-elements.